### PR TITLE
Replace survey2016 prefix with diversity in SurveyResult attrs.

### PR DESCRIPTION
**Pull request by ashercodeorg**

### CODE REVIEW

File: dashboard/test/factories.rb [hunk 1]
```diff
@@ -530,8 +530,7 @@
   factory :survey_result do
     user {create :teacher}
     kind 'Diversity2016'
-    properties {{survey2016_ethnicity_asian: "1"}}
-    properties {{survey2016_foodstamps: "3"}}
+    properties {{ethnicity_asian: "1", foodstamps: "3"}}
```
Comment (breville) [inline dashboard/test/factories.rb]: Good catch, but it should be `survey_ethnicity_asian` and `survey_foodstamps` I believe.

Comment (ashercodeorg) [reply dashboard/test/factories.rb]: Hmm. My [data fix](https://github.com/code-dot-org/code-dot-org/blob/staging/bin/oneoff/data_fix/survey_results_remove_year) removed the "survey" also. Let's discuss quickly offline.

Comment (ashercodeorg) [reply dashboard/test/factories.rb]: As discussed, we'll remove the "survey" prefix from the 2017 data.

**Comment by ashercodeorg** [comment]
PTAL.

**Comment by breville** [comment]
Updates look good.

**Comment by ashercodeorg** [comment]
And... ...  Unless you object, I'm revamping the prefix again. Given that we prefix the NPS attributes with `nps`, I think it makes sense to prefix these with `diversity`. Doing so, I think, makes: the code more readable, the attributes less ambiguous, and the redundant "survey" eliminated. Let me know if you feel otherwise.

**Comment by ashercodeorg** [comment]
While I'm here, should I change `foodstamps` to `free_and_reduced_meals` or similar?

**Comment by breville** [comment]
Both of these suggestions sound good and are fine with me.

**Comment by ashercodeorg** [comment]
And PTAL...

**Comment by breville** [comment]
Updates look good.

**Pull request merged**